hw/top_earlgrey/syn/constraints.sdc

```diff
@@ -64,6 +68,45 @@ set_output_delay ${OUT_DEL} [get_ports dio_usbdev_dp_en_o]        -clock ${USB_C
 set_output_delay ${OUT_DEL} [get_ports dio_usbdev_dn_o]           -clock ${USB_CLK_PIN}
 set_output_delay ${OUT_DEL} [get_ports dio_usbdev_dn_en_o]        -clock ${USB_CLK_PIN}
 
+#####################
+# IO clk (24MHz)    #
+#####################
+set IO_CLK_PIN clk_io_i
+set IO_TCK 40
+set_ideal_network ${IO_CLK_PIN}
+
+create_clock ${IO_CLK_PIN} -period ${IO_TCK}
+set_clock_uncertainty ${SETUP_CLOCK_UNCERTAINTY} ${IO_CLK_PIN}
+
+set IN_DEL  IO_TCK/2
+set OUT_DEL IO_TCK/2
+
+set_input_delay ${IN_DEL} [get_ports mio_in_i*]          -clock ${IO_CLK_PIN}
+
+set_output_delay ${OUT_DEL} [get_ports mio_out_o*]       -clock ${IO_CLK_PIN}
+set_output_delay ${OUT_DEL} [get_ports mio_oe_o*]        -clock ${IO_CLK_PIN}
+
+# UART RX
+set_input_delay ${IN_DEL} [get_ports dio_in_i[10]]      -clock ${IO_CLK_PIN}
+
+# UART TX
+set_output_delay ${OUT_DEL} [get_ports dio_out_o[9]]    -clock ${IO_CLK_PIN}
+set_output_delay ${OUT_DEL} [get_ports dio_oe_o[9]]     -clock ${IO_CLK_PIN}
+
+## USBDEV
+
+#####################
```

Comment:
thanks for adding this @eunchan 
I think Nuvoton is expecting to max this at 200KHz, but leaving it at 300KHz i don't think hurts :)